src/main/webapp/app/exercises/programming/shared/commits-info/commits-info.component.ts

```diff
@@ -36,23 +36,26 @@ export class CommitsInfoComponent implements OnInit, OnDestroy {
     localVC = false;
 
     ngOnInit(): void {
+        // Get active profiles, to distinguish between VC systems, and to check if localVC is enabled
+        this.profileInfoSubscription = this.profileService.getProfileInfo().subscribe((profileInfo) => {
+            this.commitHashURLTemplate = profileInfo.commitHashURLTemplate;
+            this.localVC = profileInfo.activeProfiles.includes(PROFILE_LOCALVC);
+        });
+
         if (!this.commits) {
             if (this.participationId) {
                 this.commitsInfoSubscription = this.programmingExerciseParticipationService.retrieveCommitsInfoForParticipation(this.participationId).subscribe((commits) => {
                     if (commits) {
                         this.commits = commits;
                     }
+                    this.setCommitDetails();
+                    this.groupCommits();
                 });
             }
+        } else {
+            this.setCommitDetails();
+            this.groupCommits();
         }
```

Comment:
Here is the actual bug:
A race condition.
`this.setCommitDetails()` and `groupCommits()` were called, before the commits were loaded. -> subscribe
This caused the elements in the table to not show up at all.